app/components/block_link_component.rb

```diff
@@ -16,4 +17,13 @@ def css_class
   def target
     '_blank' if new_tab
   end
+
+  def wrapper(&block)
```

Comment:
This was not as easy to implement as I'd hoped given how we use `action:` elsewhere in `ButtonComponent`. I struggled with forwarding the default block contents of this component to the custom component wrapper. Open to suggestions if I've missed something obvious.

Reply:
I can't think of any significant code changes, but maybe a clearer name? like `wrap_with_action` or something?

Reply:
> I can't think of any significant code changes, but maybe a clearer name? like `wrap_with_action` or something?

Oops, I totally forgot to address this prior to merging. Still, I think the functional behavior is pretty identical with how we want `action:` to work in other places (ButtonComponent). I actually expect there may be similar issues with buttons if we had more complex content like inner HTML. So if we update the name I'd probably want to do that consistently.